packages/media/src/calculateSrcSet.js

```diff
@@ -19,6 +19,23 @@
  */
 import aspectRatiosApproximatelyMatch from './aspectRatiosApproximatelyMatch';
 
+/**
+ * Encodes a text string as a valid Uniform Resource Identifier (URI)
+ *
+ * Only encodes strings that do not look like they're already encoded,
+ * by first looking for the percentage sign character.
+ *
+ * @param {string} str A value representing an encoded URI.
+ * @return {string} Encoded URI
+ */
+function maybeEncodeURI(str) {
```

Comment:
Should we add `replaceAll(',', '%2C')` to this function and add some unit tests?

Reply:
You mean like this?

```js
function maybeEncodeURI(str) {
  if (str.includes('%')) {
    return str.replaceAll(',', '%2C');
  }

  return encodeURI(str).replaceAll(',', '%2C');
}
```

That's a bit repetitive and less readable IMHO.

I don't see an issue with the current location of the `.replaceAll()` call.

We already have a unit test for the comma handling.